webui/src/main.tsx

```diff
@@ -72,15 +77,22 @@ class MainComponent extends React.Component<MainComponent.Props, MainComponent.S
             const user = await this.props.service.getUser();
             if (isError(user)) {
                 this.setState({ user: undefined, userLoading: false });
-            } else {
+            } else {
                 this.setState({ user, userLoading: false });
             }
         } catch (err) {
-            handleError(err);
-            this.setState({ userLoading: false });
+            this.setState({ error: handleError(err), userLoading: false });
```

Comment:
We should also set `isErrorDialogOpen: true` in this case?

Reply:
Thanks! great catch :slightly_smiling_face: